circle.yml

```diff
@@ -17,12 +17,10 @@ test:
     - make docs
 deployment:
     production:
-        branch: master
+        tag: /v*/
         commands:
-            - make docs
             - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/docs/ --delete
```

Comment:
@theseanything Sorry - I'd written this comment a while ago but hadn't submitted it. Still getting used to GH I guess.